Apps/AccountDataAccess/test/unit/PatientRepositoryTests.cs

```diff
@@ -59,28 +73,115 @@ public async Task ShouldGetDemographicsByPhn()
+        {
+            // Arrange
+            PatientDetailsQuery patientDetailsQuery = new(Hdid: Hdid, Source: PatientDetailSource.AllCache);
+
+            PatientModel? patient = null;
+
+            PatientModel cachedPatient = new()
+            {
+                Phn = Phn,
+                Hdid = Hdid,
+            };
+
+            PatientIdentityResult? patientResult = null;
+
+            PatientRepository patientRepository = GetPatientRepository(patient, patientDetailsQuery, patientResult, cachedPatient);
+
+            // Act
+            PatientQueryResult result = await patientRepository.Query(patientDetailsQuery, CancellationToken.None).ConfigureAwait(true);
+
+            // Verify
+            Assert.Equal(Phn, result.Items.SingleOrDefault()?.Phn);
+        }
+
+        /// <summary>
+        /// GetPatientIdentity by Hdid.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
+        [Fact]
+        public async Task ShouldGetPatientIdentityByHdid()
+        {
+            // Arrange
+            PatientModel expectedPatient = new()
+            {
+                Phn = Phn,
+                Hdid = Hdid,
+                Gender = Gender,
+                ResponseCode = string.Empty,
+                IsDeceased = false,
+                CommonName = new Name
+                    { GivenName = string.Empty, Surname = string.Empty },
+                LegalName = new Name
+                    { GivenName = string.Empty, Surname = string.Empty },
+            };
+
+            PatientDetailsQuery patientDetailsQuery = new(Hdid: Hdid, Source: PatientDetailSource.AllCache);
+
+            PatientModel? patient = null;
+            PatientModel? cachedPatient = null;
+
+            PatientIdentity patientIdentity = new()
+            {
+                Phn = Phn,
+                HdId = Hdid,
+                Gender = Gender,
+                HasDeathIndicator = false,
+            };
+
+            PatientIdentityResult patientIdentityResult = new(new PatientIdentityMetadata(), patientIdentity);
+
+            PatientRepository patientRepository = GetPatientRepository(patient, patientDetailsQuery, patientIdentityResult, cachedPatient);
+
+            // Act
+            PatientQueryResult actual = await patientRepository.Query(patientDetailsQuery, CancellationToken.None).ConfigureAwait(true);
+
+            // Verify
+            expectedPatient.ShouldDeepEqual(actual.Items.SingleOrDefault());
+        }
+
         /// <summary>
         /// Client registry get demographics throws api patient exception given client registry records not found.
         /// </summary>
         /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
         [Fact]
         public async Task GetDemographicsThrowsProblemDetailsExceptionGivenClientRegistryPhnNotValid()
         {
+            // Arrange
```

Comment:
What does this mean? looks like something that should be removed after completing. The TODO: tag will highlight them in Code and Rider for easier tracking.

Reply:
Updated doc comment.